lib/ffaker/identification_pl.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'date'
+
+module FFaker
+  module IdentificationPL
+    extend ModuleUtils
+    extend self
+
+    # Polish national identification number
+    # https://en.wikipedia.org/wiki/PESEL
+    def pesel
+      date = generate_valid_pesel_date
+      birthdate = pesel_birthdate(date)
+      serial_number = FFaker.numerify('####')
+      checksum = pesel_checksum(birthdate, serial_number)
+      "#{birthdate}#{serial_number}#{checksum}"
+    end
+
+    # Polish identity card number
+    # https://en.wikipedia.org/wiki/Polish_identity_card
+    def identity_card
+      letter_part = FFaker.letterify('???').upcase
+      number_part = FFaker.numerify('#####')
+      checksum = identity_card_checksum(letter_part, number_part)
+      "#{letter_part}#{checksum}#{number_part}"
+    end
+
+    alias id identity_card
+
+    # Polish driver's licence number
+    # https://en.wikipedia.org/wiki/Driving_licence_in_Poland
+    def drivers_license
+      FFaker.numerify('#####/##/####')
+    end
+
+    private
+
+    def generate_valid_pesel_date
+      from = Date.new(1800, 1, 1)
+      to = [Date.today, Date.new(2299, 12, 31)].min
+      fetch_sample([*from..to])
```

Comment:
hmm... is this casting needed?
```suggestion
      fetch_sample(from..to)
```

Reply:
Unfortunately, it's needed - `fetch_sample` performs on its argument, so after the suggested change we would receive:
`NoMethodError: undefined method `sample' for #<Range:..>`

We could probably make `fetch_sample` work with ranges, but that's probably worth a separate PR.

Reply:
oh, I see. `fetch_sample` only receiving array. 
But still It is too big array for just one date.

Reply:
I think `FFaker::Time` module could be used. For example, `FFaker::Time#between` method.

Reply:
OK, just `fetch_sample(from, to)` will be fine after #420 merged.